src/components/content-section/styles.js

```diff
@@ -0,0 +1,13 @@
+import { colors, widths } from "../../styles";
+import styled from "@emotion/styled";
+
+/** ContentSection styled component */
+const ContentDiv = styled.div({
```

Comment:
@DanielStamper You need to export the `ContentDiv` as otherwise it will be visible only in this file and nowhere else.

```suggestion
export const ContentDiv = styled.div({
```